### ISSUE TITLE
Duplicate "blacklight-modal" divs

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#698]
There are two `<div id="blacklight-modal"...>`s in each page. AMP reports the duplicate ID error. Only the first one will ever be used if they're referenced by ID, right? Can the extraneous one be removed?
